protocol/app/flags/flags.go

```diff
@@ -59,6 +78,10 @@ func GetFlagValuesFromOptions(
 		NonValidatingFullNode: DefaultNonValidatingFullNode,
 		DdAgentHost:           DefaultDdAgentHost,
 		DdTraceAgentPort:      DefaultDdTraceAgentPort,
+
+		// These are the default values from the Cosmos flags.
+		GrpcAddress: config.DefaultGRPCAddress,
```

Comment:
For my understanding, what happens when
- The validator does not pass in `grpc.addr` and `grpc.enable` in command flags
- They update these values in their `config.toml` to something that's not the default value

In this case would `flags.GrpcAddr` and `flags.GrpcEnble` different from the actual config value loaded to the app?